include/rx/ranges.hpp

```diff
@@ -1155,6 +1207,11 @@ struct ZipRange {
     constexpr size_t _size_hint(std::index_sequence<Index...>) const noexcept {
         return std::min({std::get<Index>(inputs).size_hint()...});
     }
+
+    constexpr void advance_by(size_t n) noexcept {
+        using RX_NAMESPACE::advance_by;
+        (advance_by(std::get<Inputs>(inputs), n), ...);
```

Comment:
You need to use the index version of `std::get` exactly like the `_get`, `_next`, `_size_hint` and `_at_end` methods.

Reply:
Ah yes, missed that change in the merge. :+1: